### PR TITLE
chore(HMS-2515): Add post-deploy e2e test template

### PR DESCRIPTION
**Pull request by tpapaioa**
Adding a new job template that will run end-to-end tests after deployment.

(A new saas file deployment will need to be added to app-interface for this template to take effect.)

### CODE REVIEW

File: deploy/testing-integration.yml [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+  description: "container image tag for iqe plugin"
+  value: 'hms-integration'
+- name: ENV_FOR_DYNACONF
+  value: stage_proxy
+- name: USE_BETA
+  value: "true"
+- name: IQE_PLUGINS
+  value: hms_integration
+- name: IQE_MARKER_EXPRESSION
+  value: 'host_metering'
```
Comment (mshriver) [inline deploy/testing-integration.yml:135]: I think the template default should be `tasks`

Comment (tpapaioa) [reply deploy/testing-integration.yml]: Fixed

**Review by mkholjuraev — APPROVED** [review]
LGTM!

**Comment by mkholjuraev** [comment]
Before merging can you please update the commit name to use `chore` instead of `qa`

**Comment by tpapaioa** [comment]
> Before merging can you please update the commit name to use `chore` instead of `qa`

Done. It doesn't look like I have permissions to merge the PR, though.

**Pull request merged**